sdk/deviceupdate/azure-iot-deviceupdate/src/main/java/com/azure/iot/deviceupdate/models/File.java

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+//
+
+package com.azure.iot.deviceupdate.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Map;
+
+/** Update file metadata. */
+@Fluent
+public final class File {
```

Comment:
I am worried that it might be too late for this. This would require swagger change and all sdk libraries. I take the feedback and when we create brand new sdk for GA we can take this feedback.

Reply:
For public preview we keep it.